test/stat_tracker_test.rb

```diff
@@ -62,20 +62,15 @@ def test_lowest_total_score
     assert_equal 3, @stat_tracker.lowest_total_score
   end
 
-  def test_find_home_wins
-    assert_instance_of Array, @stat_tracker.find_home_wins
-    assert_equal 2, @stat_tracker.find_home_wins.count
+  def test_find_game_teams_by_hoa_and_result
+    assert_equal 2, @stat_tracker.find_game_teams_by_hoa_and_result("home", "WIN").count
+    assert_equal 1, @stat_tracker.find_game_teams_by_hoa_and_result("away", "WIN").count
```

Comment:
Testing for `count` here is a good start! It'd be good to also test for each of these return the expected values for `@stat_tracker.find_game_teams_by_hoa_and_result("home", "WIN").first.hoa` & `#.result`, `@stat_tracker.find_game_teams_by_hoa_and_result("away", "WIN").first.hoa` & `#.result`